vendor/transaction-pool/src/graph/ready.rs

```diff
@@ -508,8 +508,12 @@ impl<Hash: hash::Hash + Member, Ex> BestIterator<Hash, Ex> {
 impl<Hash: hash::Hash + Member, Ex> sc_transaction_pool_api::ReadyTransactions
 	for BestIterator<Hash, Ex>
 {
-	fn report_invalid(&mut self, tx: &Self::Item) {
-		BestIterator::report_invalid(self, tx)
+	fn report_invalid(&mut self, _tx: &Self::Item) {
```

Comment:
I'm worried about side effects. Maybe we can do some tests using https://github.com/paritytech/polkadot-stps?tab=readme-ov-file?

Reply:
Our e2e tests on testnet get close to that (and they were showing the issue) I think we could setup something like that as well for it.